unittest/python/test_user_type.py

```diff
@@ -1,14 +1,22 @@
 import user_type
+import numpy as np
+#from packaging import version
 
 rows = 10
 cols = 20
 
 def test(mat):
-  mat.fill(mat.dtype.type(10.))
+  mat[:] = mat.dtype.type(10.)
```

Comment:
This assignment would fail for numpy version 1.17.4 which is the standard in ubuntu18.04LTS.
The bug doesn't exist anymore in 1.21.0, but I haven't checked in which version between it was fixed.
We might have to warn users about the issue when they import a custom user type, and to avoid certain assignments.